src/controllers/articleController.js

```diff
@@ -1,8 +1,67 @@
+import Article from '../models/article';
+import { ArticleServices } from '../services/articleServices';
+
 export class ArticleController {
-    // TODO Don't access database from this file you only needs
-    createArticle(req, res, next) { }
-    getAllArticles(req, res, next) { }
-    getArticle(req, res, next) { }
-    updateArticle(req, res, next) { }
-    deleteArticle(req, res, next) { }
+  // TODO Don't access database from this file you only needs
+  async createArticle(req, res, next) {
+    try {
+      const data = new Article({
+        title: req.body.title,
+        content: req.body.content,
+      });
+      const article = await ArticleServices.createArticle(data);
+      res.send(article);
+    } catch (error) {
+      res.status(404);
+      res.send({ error: 'No article created' });
```

Comment:
Replace it with `res.status(404).send({ error: 'No article created' })` and do the same in other methods